genBorder.js

```diff
@@ -0,0 +1,70 @@
+// TODO: Consider generating border classes as well that would just assign a default @neutral_silver
+// color. We need to be sure it would actually be useful before adding the extra CSS bloat.
+const _ = require("lodash");
+const fs = require("fs");
+
+
+const FILEPATH = "src/less/";
+const FILENAME = "border.less";
+const borderWidths = {
+  s: 0.0625, // 1px
+  m: 0.125, // 2px
+  l: 0.1875, // 3px
+};
```

Comment:
If the world were only ever made of rem and em units I would be happier. The ability for rem/em to scale more clearly is one of the parts that I think pixels don't do as well.